# Course material for PhD course: High throughput sequencing of non-model organisms at Nord University, Norway

## Course description
High throughput sequencing (HTS) technologies are being applied to a wide range of important topics in biology. However, the analyses of non-model organisms, for which little previous sequence information is available, pose specific problems. This course addresses the specific strengths and weaknesses of alternative HTS technologies, the computational resources needed for HTS, and how to analyze non-model species using HTS. The course consists of a practical training module, HTS bioinformatics training, and lecturing/seminars of HTS approaches specifically targeting non-model organisms. 

## Context
After a week in the lab to prepare a genome library that is sequenced on the Ion Torrent PGM, we use the second week to focus on the bioinformatics analysis of the resulting data. This involves 1) an introduction to Unix, 2) trimming and quality control, 3) de novo genome assembly, as well as 4) mapping and variant calling.
All tools, script, and programs that we use are open-source and, thus, freely availabe also after the course finishes. An introduction to Unix is invaluable for bioinformatics data analysis. It allows you to build simple pipelines, quickly extract information from large data files, and to automate processes. 
